### PR TITLE
Add source query parameter to kb autocomplete

### PR DESCRIPTION
**Pull request by hadalin**

### CODE REVIEW

File: resolwe_bio/kb/views.py [hunk 1]
```diff
@@ -89,11 +89,17 @@ class FeatureAutocompleteViewSet(HaystackViewSet):
 
     def filter_queryset(self, queryset):
         """Construct a correct filter query."""
-        query = self.request.query_params.get('query', '')
```
Comment (hadalin) [inline resolwe_bio/kb/views.py]: @kostko Any reason why empty string instead of `None`?

Comment (kostko) [inline resolwe_bio/kb/views.py]: No, probably a leftover from where the query was different.

**Comment by mstajdohar** [comment]
Missing a test for the empty query.

File: docs/contributing.rst [hunk 2]
```diff
@@ -86,6 +86,10 @@ Change directory to the Django test project::
 
     cd tests
 
+Run docker::
```
Comment (dblenkus) [inline docs/contributing.rst]: This is not part of this commit

**Comment by kostko** [comment]
BTW, there is something wrong with commit authorship.
